Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs

```diff
@@ -355,5 +359,16 @@ void UpdateImage()
 				});
 			}
 		}
+
+		void UpdateLineBreakMode()
+		{
+			AButton view = View;
+
+			if (view == null || _element == null || _renderer?.View == null)
+				return;
+
+			view.SetLineBreakMode(_element);
+			_renderer.View.SetAllCaps(_element.TextTransform == TextTransform.Default);
```

Comment:
@samhouts I used the same approach used [here](https://github.com/xamarin/Xamarin.Forms/blob/43d773d27c08ffdd43802c64676a6a670cabb7cf/Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs#L281), to preserve the text as AllCaps.